hms-api/src/service/idea_comment-service.ts

```diff
@@ -0,0 +1,130 @@
+import Uuid from '../util/Uuid';
+import NotFoundError from '../error/NotFoundError';
+import {getIdea, ideaExists} from '../repository/idea-repository';
+import {getUser, userExists} from '../repository/user-repository';
+import {
+  commentAlreadyExists,
+  getComment,
+  putComment,
+  deleteComment,
+  listComments,
+} from '../repository/idea_comment-repository';
+import ValidationError from '../error/ValidationError';
+import Comment from '../repository/domain/IdeaComment';
+import ReferenceNotFoundError from '../error/ReferenceNotFoundError';
+import CommentResponse from '../rest/Comment/CommentResponse';
+import CommentDeleteResponse from '../rest/Comment/CommentDeleteResponse';
+import CommentListResponse from '../rest/Comment/CommentListResponse';
+import ideaComment from '../repository/domain/IdeaComment';
+
+export async function createComment(
```

Comment:
there seems to be no consensus on whether it is a `Comment` or an `IdeaComment`.  It would be good to just use one everywhere